### PR TITLE
Jake/apl tests

### PR DESCRIPTION
**Pull request by jarveson**
- rip apl presets into 'json' files, so that they can be pulled into test functions and tested

### CODE REVIEW

**Comment by lologarithm** [comment]
idea to go with a PR I am cooking up, name all apl files 

XXX.apl.json -- this will let you edit those files with auto-completion once I get this readme in on how to make autocomplete work!

File: sim/deathknight/dps/dps_deathknight_test.go [hunk 1]
```diff
@@ -12,6 +14,16 @@ func init() {
 	RegisterDpsDeathknight()
 }
 
+func GetAplRotation(dir string, file string) core.RotationCombo {
```
Comment (jimmyt857) [inline sim/deathknight/dps/dps_deathknight_test.go:17]: It looks like this same function is in every test file, can it be placed in one of the testing files in `core` and shared?

Comment (jarveson) [reply sim/deathknight/dps/dps_deathknight_test.go]: good idea, moved it into test_utils

**Comment by jarveson** [comment]
@lologarithm renamed them as suggested

**Pull request merged**